pkg/pool-weighted/test/LBPool.test.ts

```diff
@@ -245,6 +245,20 @@ describe('LBPool', function () {
         .reverted;
     });
 
+    it('should not allow endTime before startTime', async () => {
```

Comment:
I think this is tested in forge, but it doesn't hurt to have it here :)

Reply:
Yes, it is, but in forge it was more like a unit test in the library, while here it tests the call in the pool.